pkg/redfish/rfcomponents.go

```diff
@@ -1204,6 +1214,10 @@ func (s *EpSystem) discoverRemotePhase1() {
 					// not sufficient.  We specify a retry count of 4 here which should be
 					// sufficient due to the exponential backoff delay in the GetRelative()
 					// function.
+					// 
+					// Because we know that node power is on here, we can check the value
+					// of powerRetryCount in later parts of code  to determine this.  It will
+					// only be set to 4 if node power is on.
 					powerRetryCount = 4
```

Comment:
The powerRetryCount variable name should be changed. (no counting is being done with the variable)
maxPowerTries

Constants could be added, but it's ok not to make if the 3 and 4 end up only being used once (which is the best option, i.e. get rid of the other references to these constants)
const maxRetriesForPowerDiscovery int := 3
const maxRetriesForPowerDiscoveryFoxconn int := 4

Reply:
Renamed to maxPowerRetries and replaced other references with a boolean.